### DISCUSSION TITLE
Issue search: how to exclude multiple terms?

### DISCUSSION THREAD

**Discussion by habibrahman29** [community/community#133845] [Projects and Issues] [answered]
### Select Topic Area

Question

### Body

Searching for issues I want to exclude assignees like this: -assignee:foo. However it seems unclear how to exclude multiple assignees. This does not work:

`-assignee:foo -assignee:bar`

**Comment by anzilparviz29** [comment, marked as answer]
Hi [andig](https://github.com/andig),

To exclude multiple assignees in a GitHub issue search, use a comma-separated list within the -assignee: filter. Here's the syntax:

![image](https://github.com/user-attachments/assets/b4229b88-0fff-41f8-8261-3b65f1cfb502)

This will exclude issues assigned to either foo or bar.
![Uploading image.png…]()
This should help you filter out issues with multiple specific assignees.

**Comment by harisdev-netizen** [comment]
Try combining multiple filters using the search bar with NOT logic. You can use something like this:

`-assignee:foo NOT -assignee:bar`

I hope this helps!
